messages/en.json

```diff
@@ -25,5 +25,7 @@
   "place-components.label.KOR": "Korea",
   "place-components.label.useCurrentLocation": "Use current location",
   "place-components.label.postalCode": "Postal Code",
-  "place-components.label.dontKnowPostalCode": "I don't know my postal code"
+  "place-components.label.dontKnowPostalCode": "I don't know my postal code",
+  "place-components.label.autocompleteAddress": "Address and number",
+  "place-components.label.autocompleteAddressFail": "No results found"
```

Comment:
@dvm1984, can you validate those translations?

![image](https://user-images.githubusercontent.com/26108090/89210661-e9a5f680-d596-11ea-8237-8574b75515a8.png)

Reply:
Address and house number fica melhor